reference_entities/refresh_records.rst

```diff
@@ -0,0 +1,40 @@
+Refresh reference entities completeness
+=======================================
+
+.. note::
+
+   Reference Entities feature is only available for the **Enterprise Edition**.
+
+Usecases
+--------
+
+Let's say we have some records having a value for each of those attributes:
+- 1 attribute of type Single Option
+- 1 attribute of type Multiple Option
+- 1 attribute of type Reference entity single link
+- 1 attribute of type Reference entity multiple links
+
+Whenever an option is removed from the Option (or Option collection) attribute, the record having the deleted option have to be refreshed in order to correctly compute the completeness.
```

Comment:
Only if it's the last one deleted (if it's a collection)

Reply:
fixed